nanostack-event-loop/platform/eventloop_config.h

```diff
@@ -19,8 +19,8 @@
 /*
  * Undefine all internal flags before evaluating the configuration
  */
-#undef NS_EVENTLOOP_USE_TICK_TIMER
-#undef NS_EXCLUDE_HIGHRES_TIMER
+//#undef NS_EVENTLOOP_USE_TICK_TIMER
```

Comment:
I think the idea was that if needed these would be defined by user provided and defined config file that gets included at the end of this file.